pyproject.toml

```diff
@@ -100,11 +100,11 @@ graphene-django-optimizer = "~0.8.0"
 #       Python 3.8+ where our direct dependency becomes part of stdlib.
 importlib-metadata = {version = "^4.4", python = "<3.8"}
 # Template rendering engine
-Jinja2 = "~3.0.3"
+Jinja2 = "~3.1.0"
```

Comment:
*Should* be okay, but note that it does remove some previously deprecated APIs (https://github.com/pallets/jinja/pull/1544)